doc/src/installation/windows-msvc.md

```diff
@@ -0,0 +1,66 @@
+# MSVC prerequistes
+
+To compile programs into an exe file, Rust requires a linker, libraries and Windows API import libraries.
+For `msvc` targets these can be acquired through Visual Studio.
+
+## Automatic install
+
+If you don't have Visual Studio already installed then [rustup-init] will offer to automatically install the prerequisites.
+Doing so means you can skip the rest of this page.
+However, it installs Visual Studio Community edition which may not be appropriate for all users.
```

Comment:
This tells folk that it might not be appropriate, but not how or where to decide (or equivalently, why it might not be appropriate). I think we can merge this even so, but it would be good to provide a little more context.